lib/options.go

```diff
@@ -327,6 +327,7 @@ type Options struct {
 	TLSCipherSuites *TLSCipherSuites `json:"tlsCipherSuites" envconfig:"K6_TLS_CIPHER_SUITES"`
 	TLSVersion      *TLSVersions     `json:"tlsVersion" ignored:"true"`
 	TLSAuth         []*TLSAuth       `json:"tlsAuth" envconfig:"K6_TLSAUTH"`
+	TLSExtraCAs     []string         `json:"tlsExtraCAs" envconfig:"K6_TLS_EXTRA_CAS"`
```

Comment:
You aren't merging the settings between the `json` and the `env`, please look at how it's done for other configurations and copy-paste them
You also don't support this to be configurable from a cli option, although I don't think that is required

Reply:
good call, I have not looked at that code before, and I need to do it